torii/src/routing.rs

```diff
@@ -136,8 +136,8 @@ enum Health {
     Healthy,
 }
 
-pub fn handle_health() -> Json {
-    reply::json(&Health::Healthy)
+pub fn handle_health() -> impl warp::Reply {
+    warp::reply::with_status("Healthy", StatusCode::OK).header("Content-Type", "text/plain")
```

Comment:
Returning a `&'static str` will be more terse and will still correctly set the `Content-Type` header: https://docs.rs/warp/latest/src/warp/reply.rs.html#436-443